pom.xml

```diff
@@ -113,7 +113,7 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
         <!-- Testing -->
-        <cobertura.version>1.9.4.1</cobertura.version>
+        <cobertura.version>2.5.2</cobertura.version>
```

Comment:
Perhaps worth using `cobertura.plugin.version` or some such, as cobertura itself is still on 2.0.3 so this might confuse people?